cfgov/ask_cfpb/templates/search/indexes/ask_cfpb/answerpage_text.txt

```diff
@@ -1,23 +1,68 @@
 {% load accent_stripper %}
+
 {{ object.short_answer | striptags }}
+
 {% for block in object.answer_content %}
-{% if block.block_type == 'text' %}
-{{ block.value.content | striptags }}
-{% endif %}{% endfor %}
-{% for block in object.answer_content %}
-{% if block.block_type == 'tip' or block.block_type == 'table' %}
-{{ block.value.content | striptags }}
-{% endif %}{% endfor %}
+    {% if block.block_type == 'text' or block.block_type == 'tip' or block.block_type == 'table' %}
+        {{ block.value.content | striptags }}
+    {% elif block.block_type == 'faq_schema' %}
+        {{ block.value.description.content }}
```

Comment:
I think `block.value.description.content` goes a bridge too far. I can get faq_schema description text to index if I change the call to `block.value.description`:


```suggestion
        {{ block.value.description }}
```